implementations/elixir/ockam/ockam_hub/test/hub/web/web_test.exs

```diff
@@ -0,0 +1,41 @@
+defmodule Test.Hub.Web.WebTest do
+  use ExUnit.Case
+  use Plug.Test
+
+  alias Ockam.Hub.Web.Router
+
+  @options Router.init([])
+
+  @http_status_ok 200
+  @http_status_bad_request 400
+  @http_status_unauthorized 401
+
+  test "create stream without authorization" do
+    conn = Router.call(conn(:post, "/streams", %{"stream_prefix" => "prefix"}), @options)
+    assert conn.status == @http_status_unauthorized
+  end
+
+  test "create stream with prefix" do
+    conn =
+      Router.call(conn(:post, create_url("/streams"), %{"stream_prefix" => "prefix"}), @options)
+
+    assert conn.status == @http_status_ok
```

Comment:
Can you add response body check as well?

Reply:
The current response body is empty, because there is no kafka running. I do not know if for unit test we should have a kafka up and running. That seems to be more related to the integration tests.

Currently the returned status code is 200, if the action is creation, I think that we should change to a 201. What do you think?

Reply:
Fair enough. Let's shift it to the integration test. Also the http codes are not ideal, so here is the update: https://github.com/ockam-network/ockam/commit/96971795d3cd280f57fbac2fc848a3a488e6a86d